uaclient/api/u/pro/unattended_upgrades/status/v1.py

```diff
@@ -0,0 +1,195 @@
+        ),
+    ]
+
+    def __init__(
+        self,
+        *,
+        systemd_apt_timer_enabled: bool,
+        apt_periodic_job_enabled: bool,
+        package_lists_refresh_frequency_days: int,
+        unattended_upgrades_frequency_days: int,
+        unattended_upgrades_allowed_origins: List[str],
+        unattended_upgrades_running: bool,
+        unattended_upgrades_disabled_reason: str,
+        unattended_upgrades_last_run: Optional[datetime.datetime]
+    ):
+        self.systemd_apt_timer_enabled = systemd_apt_timer_enabled
+        self.apt_periodic_job_enabled = apt_periodic_job_enabled
+        self.package_lists_refresh_frequency_days = (
+            package_lists_refresh_frequency_days
+        )
+        self.unattended_upgrades_frequency_days = (
+            unattended_upgrades_frequency_days
+        )
+        self.unattended_upgrades_allowed_origins = (
+            unattended_upgrades_allowed_origins
+        )
+        self.unattended_upgrades_running = unattended_upgrades_running
+        self.unattended_upgrades_disabled_reason = (
+            unattended_upgrades_disabled_reason
+        )
+        self.unattended_upgrades_last_run = unattended_upgrades_last_run
+
+
+def _get_apt_daily_job_status() -> bool:
+    try:
+        apt_daily_job_enabled = get_systemd_job_state("apt-daily.timer")
+        apt_daily_upgrade_job_enabled = get_systemd_job_state(
+            "apt-daily-upgrade.timer"
+        )
+        systemd_apt_timer_enabled = (
+            apt_daily_job_enabled and apt_daily_upgrade_job_enabled
+        )
+    except exceptions.ProcessExecutionError as e:
+        raise UnattendedUpgradesError(
+            msg=str(e),
+            msg_code="unable-to-determine-unattended-upgrade-status",
+        )
+
+    return systemd_apt_timer_enabled
+
+
+def _is_unattended_upgrades_running(
+    systemd_apt_timer_enabled: bool,
+    unattended_upgrades_cfg: Dict[str, Union[str, List[str]]],
+) -> Tuple[bool, str]:
+    if not systemd_apt_timer_enabled:
+        return (False, messages.UNATTENDED_UPGRADES_SYSTEMD_JOB_DISABLED.msg)
+
+    for key, value in unattended_upgrades_cfg.items():
+        if not value:
+            return (
+                False,
+                messages.UNATTENDED_UPGRADES_CFG_LIST_VALUE_EMPTY.format(
+                    cfg_name=key
+                ).msg,
+            )
+        if isinstance(value, str) and value == "0":
+            return (
+                False,
+                messages.UNATTENDED_UPGRADES_CFG_VALUE_TURNED_OFF.format(
+                    cfg_name=key
+                ).msg,
+            )
+
+    return (True, "")
+
+
+def _get_unattended_upgrades_last_run() -> Optional[datetime.datetime]:
+    try:
+        creation_epoch = os.path.getctime(UNATTENDED_UPGRADES_STAMP_PATH)
+    except FileNotFoundError:
+        return None
+
+    return datetime.datetime.fromtimestamp(creation_epoch)
+
+
+def unattended_upgrades_status() -> UnattendedUpgradesResult:
```

Comment:
Our pattern so far has been that the function name should be the last word in the endpoint name.
So since this endpoint is `u.pro.unattended_upgrades.status`, then this function name should be `status`

Reply:
Indeed, let me fix that